doc/user/layouts/shortcodes/replica-options.html

```diff
@@ -4,4 +4,4 @@
 `AVAILABILITY ZONE`                 | `text`     | If you want the replica to reside in a specific availability zone. You must specify an [AWS availability zone ID] in either `us-east-1` or `eu-west-1`, e.g. `use1-az1`. Note that we expect the zone's ID, rather than its name.
 `INTROSPECTION INTERVAL`            | `interval` | Default: `1s`. The interval at which to collect introspection data. See [Troubleshooting](/ops/troubleshooting) for details about introspection data. The special value `0` entirely disables the gathering of introspection data.
 `INTROSPECTION DEBUGGING`           | `bool`     | Default: `false`. Whether to introspect the gathering of the introspection data.
-`IDLE ARRANGEMENT MERGE EFFORT`     | `integer`  | The amount of effort the replica should exert on compacting arrangements during idle periods. *This is an unstable option! It may be changed or removed at any time.*
+`IDLE ARRANGEMENT MERGE EFFORT`     | `integer`  | Default: `1000`. The amount of effort the replica should exert on compacting arrangements during idle periods. The effort value is a hint indicating that work greater than or equal to an equivalent number of records being added to the arrangement should be applied per exertion cycle. Providing `NULL` disables idle arrangement compaction altogether. *This is an unstable option! It may be changed or removed at any time.*
```

Comment:
```suggestion
`IDLE ARRANGEMENT MERGE EFFORT`     | `integer`  | Default: `1000`. The amount of effort the replica should exert on compacting arrangements during idle periods. Use `NULL` to disable idle arrangement compaction altogether. *This is an unstable option! It may be changed or removed at any time.*
```

Reply:
Don't think it's helpful to include more detail, since this setting is unlikely to be used by external users.